etna/transforms/base.py

```diff
@@ -223,7 +224,11 @@ def inverse_transform(self, ts: TSDataset) -> TSDataset:
         :
             TSDataset after applying inverse transformation.
         """
-        df = ts.to_pandas(flatten=False, features=self.required_features)
+        required_features = self.required_features
+        if isinstance(required_features, list) and "target" in self.required_features:
+            features = set(ts.columns.get_level_values("feature").tolist())
+            required_features += list(match_target_quantiles(features))
```

Comment:
Don't we want to get rid of duplicates here? What if in `required_features` we already have quantiles?

Reply:
Fixed